video/out/gpu/ra.h

```diff
@@ -44,6 +44,10 @@ struct ra {
     // usually does not contain parameters required for basic functionality.
     struct ra_native_resource *native_resources;
     int num_native_resources;
+
+    // Maximum number of threads in a compute work group. Set by the RA backend
+    // at init time.
+    size_t max_compute_group_threads;
```

Comment:
maybe group this next to `max_shmem`, looks good otherwise

Reply:
Done.